src/structures/ApplicationCommand.js

```diff
@@ -216,7 +216,7 @@ class ApplicationCommand extends Base {
       name: option.name,
       description: option.description,
       required:
-        option.required ?? (stringType === 'SUB_COMMAND' || stringType === 'SUB_COMMAND_GROUP') ? undefined : false,
+        option.required ?? (stringType === 'SUB_COMMAND' || stringType === 'SUB_COMMAND_GROUP' ? undefined : false),
```

Comment:
It probably meant to do this?
```suggestion
        option.required ?? ((stringType === 'SUB_COMMAND' || stringType === 'SUB_COMMAND_GROUP') ? undefined : false),
```

Reply:
this is what I had at first but prettier removed the inner brackets

Reply:
That means they're not required, thanks for checking!